modules/op-running-pipeline-and-task-run-pods-with-privileged-security-context.adoc

```diff
@@ -0,0 +1,68 @@
+[id='op-running-pipeline-and-task-run-pods-with-privileged-security-context']
+= Running pipeline run and task run pods with privileged security context
+:context: op-running-pipeline-and-task-run-pods-with-privileged-security-context
+
+toc::[]
+
+.Procedure
+
+To run a pod (resulting from pipeline run or task run) with `privileged` security context, do the following modifications:
+
+* Configure the associated user account or service account to have an explicit SCC. You can perform the configuration using any of the following methods:
+** Use the following OpenShift command:
++
+[source, terminal]
+----
+$ oc adm policy add-scc-to-user <scc-name> -z <service-account-name>
```

Comment:
You didn't address my previous comment about this line... IMHO oc command and yaml files should do the same thing...let's choose `<service-account-name>` or `foo` format.

Reply:
Modified. PTAL.